SesameIT/jizo/ingest/parser.yml

```
@@ -0,0 +1,61 @@
+name: jizo
+pipeline:
+  - name: json_log
+    description: ""
+    filter: ""
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+
+  - name: parsed_date
+    description: ""
+    filter: ""
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_log.message.timestamp}}"
+        output_field: datetime
+        format: null
+        timezone: UTC
+
+  - name: set_base_fields
+
+stages:
+  set_base_fields:
+    actions:
+      - set:
+          source.ip: "{{json_log.message.src_ip}}"
+          "@timestamp": "{{parsed_date.datetime}}"
+          destination.ip: "{{json_log.message.dest_ip}}"
+          destination.port: "{{json_log.message.dest_port}}"
+          source.port: "{{json_log.message.src_port}}"
+          network.protocol: "{{json_log.message.app_proto}}"
+          network.transport: "{{json_log.message.proto}}"
+          action.name: "{{json_log.message.alert.action}}"
```

`event.action`  seems to be more suitable:
```suggestion
          event.action: "{{json_log.message.alert.action}}"
```

---

done